cfme/tests/middleware/test_middleware_datasource.py

```diff
@@ -131,6 +132,53 @@ def test_datasource_details(provider):
         ds_mgmt.validate_properties()
 
 
+@pytest.mark.meta(blockers=[BZ(1427141, forced_streams=["5.8", "upstream"])])
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
```

Comment:
oh yes, sorry, my bad. appliance is fixture. it won't be accessible in uncollect.